### ISSUE TITLE
Where is the real Snowfox?

### ISSUE THREAD

**Issue by ghost** [Eclipse-Community/r3dfox-old#223]
[Luminafox](https://github.com/winrevive/luminafox) was discontinued and further development happens on Snowfox. But [the Snowfox repository of the owner of Luminafox](https://github.com/winrevive/snowfox) doesn't enable Github Issues (I noted that @winrevive is not a personal account but an organization), so if there are problems I can't contact the developer for help. I searched for `Snowfox Github` on Google and it gives me various web browser projects with the name Snowfox. [This project](https://github.com/0h7z/Snowfox) has [a website](https://0h7z.com/Snowfox) that is currently 404 not found. The owner, @0h7z, is also an organization and they also don't enable Github Issues!

Long story short, where to find a r3dfox fork with DRM support? And with Github Issues, or a place to ask for support, too.

**Comment by DAPOPOBEFASTONYOAZZ** [comment]
DRM support is prohibitively costly to obtain and implement licenses and according to other submissions, this is likely not a feature coming and if it is, it will not happen any time soon.

**Comment by ghost** [comment]
I don't ask r3dfox to support DRM. But Luminafox is what the r3dfox recommends people wanting DRM support to use. Now Luminafox is defunct with Snowfox is the successor. But I have many questions about Snowfox. Where is the real development happening? Where to ask for support?

**Comment by ghost** [comment]
Perhaps I have overlooked it. But the @winrevive organization has a website hosted on github.io, and this website does mention a discord channel.

https://winrevive.github.io/

The @0h7z organization's website link for their Snowfox mentioned on their Github repository, https://0h7z.com/Snowfox, is incorrect. It must be https://0h7z.com/en/snowfox/. The [development repository](https://github.com/Heptazhou/Snowfox) does enable Github Issues.

**Comment by DAPOPOBEFASTONYOAZZ** [comment]
Ah I understand, I apologize I misunderstood your original question. Hopefully that will lead you toward the correct path.
